### PR TITLE
[HAR-223] Add code block collapse control

### PR DESCRIPTION
**Pull request by romypetitjean**
Issue key: [HAR-223]

## Summary of your proposal
Enables collapsing code blocks and improves readability of example.html

## Proposed changes in this pull request:

* Add collapsible styling to docs.html as this styling does not belong to the UI-library style
* Add collapsible functionality to example.html to enable collapsing code blocks
* Improve readability of example.html

### Feature

* Enables collapsing code blocks

### CODE REVIEW

File: site/_includes/example.html [hunk 1]
```diff
@@ -3,34 +3,57 @@
...
+{% capture code-example %}
+	{%- highlight html -%}
+		{{- include.content -}}
+	{%- endhighlight -%}
+{% endcapture %}
+
+<div class="layout__section layout__section--spaced-70 layout__section--fitted layout__section--bordered layout__section--rounded">
+
+	<!-- Default example including visual example and code example -->
+	{%- if include.show-code-example-only == null and include.show-visual-example-only == null -%}
```
Comment (edgargressie) [inline site/_includes/example.html]: Not sure if I like the show-only syntax. When you display everything by default, it feels more logic to hide something.

File: site/_includes/example.html [hunk 2]
```diff
@@ -3,34 +3,57 @@
...
-			{%- if include.hide_markup == null -%}
-				{% assign layoutClasses="layout__section layout__section--spaced-0 layout__section--rounded-bottom" %}
-				{%- if include.hide_preview == null -%}
-					{% assign layoutClasses="layout__section layout__section--spaced-0 layout__section--rounded-bottom layout__section--bordered-top" %}
-				{%- endif -%}
-				<div
-					class="{{ layoutClasses }}"
-				>
-					{%- highlight html -%}
-						{{- include.content | replace: 'data-src="holder.js', 'src="...' -}}
```
Comment (edgargressie) [inline site/_includes/example.html]: Is it correct that the replace pipe is deleted?

File: site/docs/layouts/page.md [hunk 3]
```diff
@@ -33,5 +33,5 @@ page
 {% endcapture %}
 {% include example.html
 	content=page
-	hide_preview=true
+	show-code-example-only=true
```
Comment (stevengunneweg) [inline site/docs/layouts/page.md]: Name could be improved I think

File: site/_includes/example.html (hunk 2 above)
Comment (romypetitjean) [reply site/_includes/example.html]: Yes that's correct, we don't use this

**Pull request merged**